src/PlusDataCollection/Andor/vtkPlusAndorVideoSource.cxx

```diff
@@ -468,6 +468,10 @@ PlusStatus vtkPlusAndorVideoSource::SetBiasCorrectionImage(const std::string bia
   try
   {
     cvBiasCorrection = cv::imread(biasFilePath, cv::IMREAD_GRAYSCALE);
```

Comment:
cv::imread throws exceptions, so this might not be necessary. Have you run into a case where this is needed?

Reply:
Yes, running with the bias/flat paths set to a path that can't be found will silently fail the imread, and the resulting cv::Mat will be empty. This is just for easier debugging in that case